src/resmom_win/start_exec.c

```diff
@@ -2481,6 +2490,7 @@ start_process(task *ptask, char **argv, char **envp)
 	char			**env;
 	char			*env_str = NULL;
 	char			*p = NULL;
+	char			cmd_shell[MAX_PATH + 1] = {0};
```

Comment:
I'm surprised the compiler doesn't issue a warning about this. Please initialize with {'\0'}.